ListApp/Sources/ListApp.Api.Services/RouteHelper/RouteHelperBootstrapper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ListApp.Contracts.Interfaces;
+using Microsoft.Practices.Unity;
+
+namespace ListApp.Api.Services.RouteHelper
+{
+    public class RouteHelperBootstrapper : IUnityContainerBootstrapper
```

Comment:
Is this a good idea though? Imagine a situation when the Services assembly contains more classes than just RouteHelper (eg. an GuidGenerator). If there was only one bootstrapper for entire assembly, whenever using any service out of the assembly, all implementation of all services would register. What if wanted to use the provided RouteHelper implementation, but implement GuidGenerator service on my own?

Reply:
In that case, I'm guessing you should have an assembly for RouteHelper and an assembly for GuidGenerator. Otherwise, you'd bring to your application more than you need in the case you want your own GuidGenerator implementation. Of course, you can overwrite the registration in DI container, but that's not very nice, as you'd probably guess. Look at an assembly as a logical unit of high-level functionality you bring to your application. The bootstrapper is sort of an entry point to it.